src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,161 @@
+.contact {
+  background-color: $background-black;
+
+  &__container {
+    display: grid;
+    gap: 15px;
+
+    @include main-paddings;
+
+    @include max-size;
+  }
+
+  &__subtitle {
+    font-size: 14px;
+    color: $contrast-color;
+  }
+
+  &__list {
+    display: none;
+  }
+
+  &__link {
+    transition: color 0.3s;
+    &:hover {
+      color: $contrast-color;
+    }
+  }
+
+  &__form {
+    display: grid;
+    gap: 5px;
+
+    &-item {
+      display: grid;
+      grid-column: 1 / 3;
+    }
+  }
+
+  &__input {
+    @include main-input;
+    margin-bottom: 10px;
+    order: 2;
+
+    &:-webkit-autofill,
+    &:-webkit-autofill:hover,
+    &:-webkit-autofill:focus {
+      @include main-input;
+      -webkit-text-fill-color: $white;
+      transition: background-color 5000s ease-in-out 0s;
+    }
```

Comment:
Can you set it for all inputs ?
Or create the mixin and put the color as a param